packages/perseus-core/src/data-schema.ts

```diff
@@ -406,8 +406,7 @@ export type PerseusImageBackground = {
     // The scale of the image
     // NOTE: perseus_data.go says this is required, but nullable, even though
     // it isn't necessary at all.
-    // Yikes, production data as this as both a number (1) and string ("1")
-    scale?: number | string;
+    scale?: number;
```

Comment:
We convert `scale` to a number in the parser now, so this type awkwardness can go away.

Reply:
So is `data-schema` now the representation of the _parsed_ JSON? Because it was the representation of the _stored_ JSON, so the parser wouldn't affect that right?

Reply:
`data-schema` _should_ represent the _latest_ version of the stored JSON. The purpose of the parser is to migrate past versions of the JSON to match what's in data-schema. Really, data-schema never represented "the" stored JSON, because we have multiple versions of JSON stored.